Controllers/Backend/ConnectBaseController.php

```diff
@@ -1767,11 +1767,9 @@ protected function getSnHttpClient()
      */
     protected function getHost()
     {
-        $host = $this->getConfigComponent()->getConfig('connectDebugHost', 'connect.shopware.com');
-        if ($host) {
-            $host = $this->getConfigComponent()->getSocialNetworkPrefix() . $host;
-        } else {
-            $host = $this->getConfigComponent()->getSocialNetworkPrefix() . $this->getConfigComponent()->getMarketplaceUrl();
+        $host = $this->getConfigComponent()->getConfig('connectDebugHost', Config::MARKETPLACE_URL);
+        if (!$host) {
+            $host = $this->getConfigComponent()->getMarketplaceUrl();
```

Comment:
Ok my fault. It can be empty if connectDebugHost is an empty string in DB.
Why you don't use
```
$host = $this->getConfigComponent()->getConfig('connectDebugHost');
if (!$host || $host == "") {
$host = $this->getConfigComponent()->getMarketplaceUrl();
}
```

See getMarketplaceUrl what actually does:

```
     public function getMarketplaceUrl()
     {
         if ($this->getConfig('marketplaceNetworkUrl')) {
             return $this->getConfig('marketplaceNetworkUrl');
         }
 
          return self::MARKETPLACE_URL;
      }
```

If marketplaceNetworkUrl doesn't exists in DB, it will return ```self::MARKETPLACE_URL```